--- /opt/conda/lib/python3.11/http/cookies.py	2024-02-16 21:20:27.000000000 +0000
+++ /opt/conda/lib/python3.11/http/cookies.mod.py	2024-03-13 22:38:10.000000000 +0000
@@ -282,9 +282,10 @@
         "httponly" : "HttpOnly",
         "version"  : "Version",
         "samesite" : "SameSite",
+        "partitioned" : "Partitioned",
     }
 
-    _flags = {'secure', 'httponly'}
+    _flags = {'secure', 'httponly', 'partitioned'}
 
     def __init__(self):
         # Set defaults
