--- /usr/lib/python3.10/http/cookies.py	2023-11-20 15:14:05.000000000 +0000
+++ /usr/lib/python3.10/http/cookies.mod.py	2024-03-14 08:53:23.379289973 +0000
@@ -282,9 +282,10 @@
         "httponly" : "HttpOnly",
         "version"  : "Version",
         "samesite" : "SameSite",
+        "partitioned" : "Partitioned",
     }
 
-    _flags = {'secure', 'httponly'}
+    _flags = {'secure', 'httponly', 'partitioned'}
 
     def __init__(self):
         # Set defaults
